include/NovelRT/DotNet/RuntimeService.h

```diff
@@ -0,0 +1,51 @@
+// Copyright © Matt Jones and Contributors. Licensed under the MIT Licence (MIT). See LICENCE.md in the repository root for more information.
+
+#ifndef NOVELRT_RuntimeService_H
+#define NOVELRT_RuntimeService_H
+
+#ifndef NOVELRT_H
+#error Please do not include this directly. Use the centralised header (NovelRT.h) instead!
+#endif
+
+namespace NovelRT::DotNet {
+  class RuntimeService {
+  private:
+    NovelRT::Utilities::Lazy<hostfxr_handle> _hostContextHandle;
```

Comment:
Base namespace is the same; you could just use `Utilities::Lazy<T>` instead. This applies to the others too. 😄